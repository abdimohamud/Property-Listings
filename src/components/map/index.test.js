import * as React from 'react'
import renderer from 'react-test-renderer'

import Map from '.'

test('renders correctly', () => {
  const address = 'Baker street'
  const tree = renderer.create(<Map address={address} />).toJSON()
  expect(tree).toMatchSnapshot()
})
