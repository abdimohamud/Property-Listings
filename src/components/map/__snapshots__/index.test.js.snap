// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mapouter"
>
  <div
    className="gmap_canvas"
  >
    <iframe
      frameBorder="0"
      height="500"
      id="gmap_canvas"
      marginHeight="0"
      marginWidth="0"
      scrolling="no"
      src="https://maps.google.com/maps?q=Baker%20street&t=&z=16&ie=UTF8&iwloc=&output=embed"
      title="Baker street"
      width="100%"
    />
  </div>
</div>
`;
